ansible/roles/dof_etherpad/templates/deployment.yml.j2

```
@@ -67,11 +67,24 @@ spec:
           name: apikey
         resources:
           limits:
-            cpu: {{ ETHERPA_CPU_LIMITS|default("1000m", true) }}
-            memory: {{ ETHERPA_MEMORY_LIMITS|default("1Gi", true) }}
+            cpu: {{ ETHERPAD_CPU_LIMITS|default("1000m", true) }}
+            memory: {{ ETHERPAD_MEMORY_LIMITS|default("1Gi", true) }}
           requests:
-            cpu: {{ ETHERPA_CPU_REQUESTS|default("100m", true) }}
-            memory: {{ ETHERPA_MEMORY_REQUESTS|default("128Mi", true) }}
+            cpu: {{ ETHERPAD_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ ETHERPAD_MEMORY_REQUESTS|default("128Mi", true) }}
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 9001
+          initialDelaySeconds: 60
+          periodSeconds: 10
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 9001
+          path: /health
+          periodSeconds: 10
+          failureThreshold: 3
```

I woud thing it's better to have an shor redinessProb who fail after an short time with an failure Threshold like 2 or 3 and the livenessProbe like an x amout of the redinessProb.
```suggestion
        readinessProbe:
          httpGet:
            path: /health
            port: 9001
          initialDelaySeconds: 60
          periodSeconds: 5
          failureThreshold: 2
        livenessProbe:
          httpGet:
            path: /health
            port: 9001
          path: /health
          periodSeconds: 10
          failureThreshold: 3
```